src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -159,13 +160,63 @@
           />
   </Target>
 
+  <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+    <CommonPriFile Condition="'$(CommonPriFile)' == ''">$(RuntimePath)/resw/resources.pri</CommonPriFile>
+    <TestSpecificPriFile>$(TestPath)\resources.pri</TestSpecificPriFile>
+  </PropertyGroup>
+
+
+
+  <Target Name="CalculateResWFiles" Condition="'$(BuildingUAPVertical)' == 'true'">
```

Comment:
When building a test this target is loaded at the same time as resources.targets, so we could hit the case where this test have an embedded resource but since the global scope is calculated on load time and no in execution time by the targets the files wouldn't be updated and will not contain that new resw that was just created by resources.targets. So we need to put it inside a target so that this target is executed right before creating the pri file which means resources.targets was already executed and it will have all the necessary files.